src/gui/wizard/flow2authwidget.cpp

```diff
@@ -113,7 +113,7 @@ void Flow2AuthWidget::slotAuthResult(Flow2Auth::Result r, const QString &errorSt
     connect(fetchUserNameJob, &JsonApiJob::jsonReceived, this, [this, fetchUserNameJob, r, errorString, user, appPassword](const QJsonDocument &json, int statusCode) {
         fetchUserNameJob->deleteLater();
         if (statusCode != 100) {
-            qCWarning(lcFlow2AuthWidget) << "Could not fetch username";
+            qCWarning(lcFlow2AuthWidget) << "Could not fetch username. dav_user will maybe set to a wrong value";
```

Comment:
instead set dav_user to an empty string and keep the log message as it was before